www/src/components/Sponsors.astro

```diff
@@ -0,0 +1,33 @@
+<style lang="scss">
+.wrapper {
+  margin-bottom: 3rem;
+  margin-top: 3rem;
+}
+
+.list {
+  align-items: center;
+  display: flex;
+  justify-content: center;
+
+  a {
+    &::before,
+    &::after {
+      display: none;
+    }
+  }
+}
+
+.title {
+  font-size: 0.75em;
+  font-weight: 700;
+  text-align: center;
+  margin-bottom: 1.5rem;
+}
+</style>
+
+<div class="wrapper">
+  <h2 class="title">Sponsors</h2>
```

Comment:
Can this be "Sponsored by" or "Proudly sponsored by"? Also, should there be some link to our open collective here?

Reply:
![Screen Shot 2021-09-16 at 10 37 50](https://user-images.githubusercontent.com/1369770/133651333-1eb5ec11-99e7-40b4-aced-635a5dcb1ebf.png)

Reply:
How is this looking?